dtls.c

```diff
@@ -3531,10 +3557,13 @@ check_certificate_request(dtls_context_t *ctx,
     current_sig_alg = dtls_uint8_to_int(data);
     data += sizeof(uint8);
 
+    data_length += sizeof(uint16);
```

Comment:
I presume this needs to be `data_length -= ...`.

Reply:
Yes.